app/containers/RoomItem/useRowHeight.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { createContext, useContext } from 'react';
+import { PixelRatio } from 'react-native';
+
+interface IRowHeightProvider {
+	children: React.ReactNode;
+}
+
+interface IRowHeightContextProps {
+	ROW_HEIGHT: number;
+	ROW_HEIGHT_CONDENSED: number;
+}
+
+const RowHeightContext = createContext<IRowHeightContextProps>({} as IRowHeightContextProps);
+
+export const RowHeightProvider = ({ children }: IRowHeightProvider) => {
+	const RowHeight = PixelRatio.getFontScale();
+	const ROW_HEIGHT = 75 * RowHeight;
+	const ROW_HEIGHT_CONDENSED = 60 * RowHeight;
```

Comment:
Previously when these values were actual constants, it made sense to have them uppercase. Now they're not, they should be camelCase.